src/imgui/ImGuiSymbols.cc

```diff
@@ -158,11 +204,32 @@ static void drawTable(ImGuiManager& manager, const SymbolManager& symbolManager,
 
 			if (ImGui::TableNextColumn()) { // name
 				im::ScopedFont sf(manager.fontMono);
-				ImGui::TextUnformatted(sym.name(symbolManager));
+				im::StyleColor(sym.segment(symbolManager) && *sym.segment(symbolManager) > 0, ImGuiCol_Text, getColor(imColor::YELLOW), [&]{
+					ImGui::TextUnformatted(sym.name(symbolManager));
+				});
+				auto symNameMenu = tmpStrCat("symbol-manager##", sym.name(symbolManager)).data();
+				if (ImGui::IsItemClicked(ImGuiMouseButton_Right)) {
+					ImGui::OpenPopup(symNameMenu);
+				}
+				im::Popup(symNameMenu, [&]{ drawContext(motherBoard, sym); });
```

Comment:
We discussed on IRC to try to rework this so that right-click in the whole row work, not only on the text item itself.